hooks/00-webhook.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+#
+# Use http request to send notification to Telegram chat
+
+set -euo pipefail
+IFS=$'\n\t'
+
+TXT="${1}"
```

Comment:
Upper case in var names is reserved for env vars not a local ones.

Reply:
Fixed